### DISCUSSION TITLE
chrome.tabs.move() on active tab does not scroll vertical tabs

### DISCUSSION THREAD

**Discussion by vanowm** [microsoft/MicrosoftEdge-Extensions#125] [Q&A]
When there are many opened tabs that require scroll and `chrome.tabs.move()` used on active tab, vertical tabs are not automatically scrolled to make sure activate tab is still visible.

Minimizing and then restore the window brings active tab into view.

Is there a way bring a tab into view after it was moved from browser extension?

**Comment by ManikanthMSFT** [comment]
Hey @vanowm, apologies for the delay in responding to your query.

We are looking into this and will let you know as soon as we have an update.

Thank you for your patience.

**Comment by sandeepchads** [comment]
@vanowm currently, this issue is low priority on our backlog. We would love to understand the importance of this bug for you.  Also it would help to understand what scenario is not working due to this issue.

https://developer.chrome.com/docs/extensions/reference/api/tabs#move

**Comment by vanowm** [comment]
My use case is extension that allows user pick where a new tab should be opened: left/right of current tab, or first/last. So, when a tab was moved to a correct position, but it's not visible due to scrolling issue, this ruins user's experience.

**Reply by ManikanthMSFT** [reply]
Hi @vanowm, thank you for your feedback. Currently this item is not prioritized on our backlog. We will update this thread when we consider this.

**Comment by sandeepchads** [comment]
@vanowm - can you please share a sample extension with repro steps where I can see this issue happening? That will help us better understand the impact of this.
